org.civicrm.reqangular/js/test/test-main.js

```diff
@@ -1,10 +1,10 @@
 /* globals CRM */
 
-var TEST_REGEXP = /(spec|test)\.js$/i;
+var TEST_REGEXP = /spec\.js$/i;
 var allTestFiles = [];
 var extPath = '/base/tools/extensions/civihr/org.civicrm.reqangular';
-var mocksPath = extPath + '/test/mocks';
-var srcPath = extPath + '/src/common';
+var mocksPath = extPath + '/js/test/mocks';
```

Comment:
I think it may be safer to use `Path.join()` here

Reply:
this file is loaded in the browser when karma is launched, it's not executed in node, so `require('path')` won't work